Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -123,8 +123,35 @@ function validate_cards() {
         SetTestStateAborted
         return 1
     fi
+    echo "****************************" >> TestExecution.log
```

Comment:
LogMsg "****************************" will reach the same goal.

Reply:
Done